docs/rfc/luthier-design.md

```diff
@@ -186,38 +186,41 @@ value in Druid, but still perform human friendly searches in Fili (i.e.
 
 A search provider's configuration is a table with at least one key:
 
-    * type - The type of the search provider. This is used by Fili to find the
-        right [SearchProvider Factory](#search-provider-factory). The name
-        doesn't matter, except search provider types exist in a global
-        namespace, so care should be taken to make sure the names are unique.
+* type - The type of the search provider. This is used by Fili to find the
+    right [SearchProvider Factory](#search-provider-factory). The name
+    doesn't matter, except search provider types exist in a global
+    namespace, so care should be taken to make sure the names are unique.
 
 It may support any other keys that particular search provider requires to be built.
 
 Fili supports the following built in SearchProviders:
 
-    * fili-lucene - Uses
-        [Apache Lucene](http://lucene.apache.org/) to handle field resolution.
-        Additional keys:
-            * luceneIndexPath - A String. The path to the lucene index files
-            * maxResults - An integer. The maximum number of results allowed on a single
-                page of results.
-            * searchTimeout - An integer. The maximum number of milliseconds Lucene should
-                search before timing out. Defaults to the
-                `lucene_search_timeout_ms` configuration parameter.
-
-    * fili-scan - Uses an in-memory data structure to handle field resolutions.
-        Not recommended for large dimensions.
-        Additional keys:
-            None
-
-    * fili-noop - Does not do any field resolution at all. Useful for dimensions
-        that don't have any fields.
-        Additional keys:
-            * approximateCardinality - An integer. The rough size of the dimension.
-                This is used when Fili is deciding whether to send a
-                weight-check query before sending an actual query. If not
-                present, Fili will use the query_weight_limit configuration
-                parameter.
+* fili-lucene - Uses
+    [Apache Lucene](http://lucene.apache.org/) to handle field resolution.
+
+    Additional keys:
```

Comment:
Should this list be unindented as well?

Reply:
A tab-indented single star list will be automatically rendered as a sublist (equiv to **) 
I think unifying style in all .md in our repo might be potentially interesting thing to do, but for this reformatting, I will make all sublist using tab + *

This is a render of what it looks like, which I think is reasonable: 
![Screen Shot 2019-05-30 at 6 38 42 PM](https://user-images.githubusercontent.com/24768277/58672028-a30d9a00-830a-11e9-816c-758cf1c9d968.png)

Reply:
👍